config/locales/en.yml

```diff
@@ -1164,6 +1171,15 @@ en:
       update_assessment: Update assessment
     update:
       success: Recommendation was successfully reviewed.
+  reconsult_consultees_component:
+    date_label: Consultee reponse required by
+    formatting_hint_html: The message is formatted using <a href="https://www.notifications.service.gov.uk/using-notify/formatting">GOV.UK Notify formatting rules</a>. Placeholders like <nobr><code>%{closing_date}</code></nobr> will be substituted when the email is sent.
```

Comment:
"Placeholders like" — should we make sure there's an actual list of the supported placeholders somewhere? Not a blocker, though.

Reply:
Ideally, we'd link 'placeholders' to a documentation page but I don't think we have any in-app documentation at the moment so didn't want to expand the story.

Reply:
Yeah, it's more something to follow up on later.